pages/Login/signIn/index.tsx

```diff
@@ -0,0 +1,22 @@
+import style from "@/pages/login/login.module.scss";
+import SocialSign from "@/src/components/pageComponent/login/socialSign";
+import LoginForm from "@/src/components/pageComponent/login/loginForm";
+import LoginMenu from "@/src/components/pageComponent/login/loginMenu";
+function SignIn() {
+  return (
+    <div className={style.loginPage}>
+      <section className={style.loginBox}>
+        <header className={style.header}>
+          <LoginMenu signState={true} />
```

Comment:
컴포넌트의 props로 boolean값을 넘길때는 이렇게 표현해주는게 일반적입니다~
```suggestion
          <LoginMenu signState />
```

Reply:
처음 알았네요 감사합니다!